shared/js/project.js

```diff
@@ -125,6 +135,12 @@ var projects = module.exports = {
     }
   },
 
+  showAdmin: function() {
+    if (typeof dashboard.admin.showProjectAdmin === 'function') {
```

Comment:
paranoia? not being in dashboard? I think at some point in the evolution of this code, the admin stuff was only included for admins, not for everyone, so it needed this to work for non-admins.

Do you think I should remove the check?